Gemfile.lock

```diff
@@ -237,72 +238,72 @@ GEM
       timeout
     net-protocol (0.1.3)
       timeout
-    net-scp (3.0.0)
-      net-ssh (>= 2.6.5, < 7.0.0)
+    net-scp (4.0.0.rc1)
+      net-ssh (>= 2.6.5, < 8.0.0)
     net-smtp (0.3.1)
       digest
       net-protocol
       timeout
-    net-ssh (6.1.0)
+    net-ssh (7.0.1)
     nio4r (2.5.8)
-    nokogiri (1.13.3-x86_64-darwin)
+    nokogiri (1.13.8-x86_64-darwin)
       racc (~> 1.4)
     orm_adapter (0.5.0)
     pagy (5.10.1)
       activesupport
     parallel (1.22.1)
-    parser (3.1.1.0)
+    parser (3.1.2.1)
       ast (~> 2.4.1)
-    pg (1.3.5)
+    pg (1.4.3)
     propshaft (0.6.4)
       actionpack (>= 7.0.0)
       activesupport (>= 7.0.0)
       rack
       railties (>= 7.0.0)
-    public_suffix (4.0.6)
-    puma (5.6.4)
+    public_suffix (5.0.0)
+    puma (5.6.5)
       nio4r (~> 2.0)
     racc (1.6.0)
-    rack (2.2.3)
-    rack-test (1.1.0)
-      rack (>= 1.0, < 3)
-    rails (7.0.2.3)
-      actioncable (= 7.0.2.3)
-      actionmailbox (= 7.0.2.3)
-      actionmailer (= 7.0.2.3)
-      actionpack (= 7.0.2.3)
-      actiontext (= 7.0.2.3)
-      actionview (= 7.0.2.3)
-      activejob (= 7.0.2.3)
-      activemodel (= 7.0.2.3)
-      activerecord (= 7.0.2.3)
-      activestorage (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    rack (2.2.4)
+    rack-test (2.0.2)
+      rack (>= 1.3)
+    rails (7.0.4)
+      actioncable (= 7.0.4)
+      actionmailbox (= 7.0.4)
+      actionmailer (= 7.0.4)
+      actionpack (= 7.0.4)
+      actiontext (= 7.0.4)
+      actionview (= 7.0.4)
+      activejob (= 7.0.4)
+      activemodel (= 7.0.4)
+      activerecord (= 7.0.4)
+      activestorage (= 7.0.4)
+      activesupport (= 7.0.4)
       bundler (>= 1.15.0)
-      railties (= 7.0.2.3)
+      railties (= 7.0.4)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
-    rails-html-sanitizer (1.4.2)
+    rails-html-sanitizer (1.4.3)
       loofah (~> 2.3)
-    railties (7.0.2.3)
-      actionpack (= 7.0.2.3)
-      activesupport (= 7.0.2.3)
+    railties (7.0.4)
+      actionpack (= 7.0.4)
+      activesupport (= 7.0.4)
       method_source
       rake (>= 12.2)
       thor (~> 1.0)
       zeitwerk (~> 2.5)
     rainbow (3.1.1)
     rake (13.0.6)
-    ransack (3.0.0)
-      activerecord (>= 6.0.4)
-      activesupport (>= 6.0.4)
+    ransack (3.2.1)
+      activerecord (>= 6.1.5)
+      activesupport (>= 6.1.5)
       i18n
-    rb-fsevent (0.11.1)
+    rb-fsevent (0.11.2)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
-    redis (4.6.0)
-    regexp_parser (2.2.1)
+    redis (4.8.0)
```

Comment:
### [redis](https://github.com/redis/redis-rb)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 4.6.0 → 4.8.0

[_[change-log](https://github.com/redis/redis-rb/blob/master/CHANGELOG.md), [source-code](https://github.com/redis/redis-rb/tree/v4.8.0), [gem-diff](https://my.diffend.io/gems/redis/4.6.0/4.8.0)_]





<details>
<summary>Commits</summary>

A change of **58** commits. See the full changes on [the compare page](https://github.com/redis/redis-rb/compare/v4.6.0...v4.8.0).

These are the first 10 commits:
- (115b70b) [fix typo in changelog](https://github.com/redis/redis-rb/commit/115b70bbc1d9b5e84ea6c6550a74e91cbf6d7793)
- (21ec1de) [Merge pull request #1068 from nikokon/master](https://github.com/redis/redis-rb/commit/21ec1dec064f43def1f47983431861e6491b770e)
- (7ff46b6) [Fix support for transaction inside pippelines](https://github.com/redis/redis-rb/commit/7ff46b64ecae57063584104a34e7cec89412b29c)
- (cc21d67) [Merge pull request #1073 from casperisfine/transactions-in-p…](https://github.com/redis/redis-rb/commit/cc21d6762a63737969aa01139bb071da1f710b74)
- (159eb79) [Improve error message for initial setup of cluster mode](https://github.com/redis/redis-rb/commit/159eb79f53167387b166008419c442294ef4796d)
- (13c7a8e) [Merge pull request #1075 from supercaracal/improve-err-msg-f…](https://github.com/redis/redis-rb/commit/13c7a8e4fdb23d0fde8534ea974fdf089c8d11bc)
- (3c7da2b) [Fix a option parsing bug for cluster mode that percent encod…](https://github.com/redis/redis-rb/commit/3c7da2b5c7d830e699bca5c2611c25fe0f8e2299)
- (86df222) [remove hiredis ssl not implemented exception](https://github.com/redis/redis-rb/commit/86df222e80e813127592d9bc9e0bc463a3d11a0d)
- (1262e75) [remove hiredis test for ssl option](https://github.com/redis/redis-rb/commit/1262e75c00ac4390b6ccdbc70f72dd7b16602398)
- (8650985) [Merge pull request #1076 from supercaracal/prevent-double-un…](https://github.com/redis/redis-rb/commit/86509850818ffe6d8917acb325be4cedf7fd9873)


</details>